lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -48,12 +48,16 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
         self.logger = logger
         self.uri_prefix = config.rest_uri
         self.commit_id = config.COMMIT_ID
+        self.server_config = config
 
     def get(self):
         """
         Return server configuration information required by web clients
         including the Pbench dashboard UI. This includes:
 
+        authentication: A JSON object containing the authentication parameters
+                        required for the web client to authenticate the end-user
+                        with OIDC broker.
```

Comment:
Perhaps "required for the web client to use OIDC authentication"?  (It doesn't _have_ to be a "broker"....)